### PR TITLE
stratify fold assigment when doing dataset generator, add fold column…

### PR DESCRIPTION
**Pull request by kevin-yauris**
… when using modeller

### CODE REVIEW

File: poc/configs/dataset_generator_config.py [hunk 1]
```diff
@@ -16,5 +16,6 @@
     ),
     train_ratio=0.7,
     label_column=LABEL,
-    strata_column=Columns.VENDOR_ID
+    strata_column=Columns.VENDOR_ID,
+    stratify_train_n_fold=5
```
Comment (falloncandra) [inline poc/configs/dataset_generator_config.py:20]: mau be we should put this in  DefaultValuesSettings constant

File: poc/modules/dataset_generator.py [hunk 2]
```diff
@@ -76,6 +81,22 @@ def stratified_sample_by_strata_column(
 
         return sampled_df, unsampled_df
 
+    @staticmethod
+    def add_stratified_fold_column(df: DataFrame, strata_column: str, fold_column: str, n_folds: int,
```
Comment (falloncandra) [inline poc/modules/dataset_generator.py:85]: I think this should be private method

File: poc/modules/dataset_generator.py [hunk 3]
```diff
@@ -76,6 +81,22 @@ def stratified_sample_by_strata_column(
...
+                                   seed: int = 16) -> DataFrame:
```
Comment (falloncandra) [inline poc/modules/dataset_generator.py:86]: hm i think it's better to use DefaultValues.SEED in general constant

File: poc/configs/dataset_generator_config.py (hunk 1 above)
Comment (kevin-yauris) [reply poc/configs/dataset_generator_config.py:20]: done, I put it in general_constants.DefaultValues

File: poc/modules/dataset_generator.py (hunk 2 above)
Comment (kevin-yauris) [reply poc/modules/dataset_generator.py:85]: done

File: poc/modules/dataset_generator.py (hunk 3 above)
Comment (kevin-yauris) [reply poc/modules/dataset_generator.py:86]: agree, done

**Pull request merged**